measurements/api/measurements.py

```diff
@@ -244,19 +323,25 @@ def list_measurements(
     iter_start_time = time.time()
     results = []
     for row in q:
+        measurement_id = '{}-{}'.format(MSM_ID_PREFIX, row.msm_no)
         url = urljoin(
             current_app.config['BASE_URL'],
-            '/api/v1/measurement/%s' % row.m_id
+            '/api/v1/measurement/%s' % measurement_id
         )
         results.append({
             'measurement_url': url,
-            'measurement_id': row.m_id,
+            'measurement_id': measurement_id,
```

Comment:
IMHO we should not override `measurement_id` field here according to principle of least astonishment. There is already entity named `measurement_id` and it's user-generated.

Reply:
Following the principle of least astonishment, I would expect that I can use `measurement_id` as some identifying to identify measurements, that is a thing I cannot do.

IMHO we should be transforming this and adding it on the pipeline stage. See: https://github.com/TheTorProject/ooni-pipeline/issues/48#issuecomment-328652712. For lack of that I would say we rewrite it on the measurements side for consistency purposes.

Reply:
> I would expect that I can use measurement_id as some identifying to identify measurements

Well, it's possible. You can use it to identify some set of measurements (usually -- one measurement).

> that is a thing I cannot do

I'd rather drop `measurement_id` from results then. Or just name it differently. I mean, there is entity named `measurement_id` in measurement json blob and it's there till last "old" ooniprobe dies, so we can't easily rename it.

Values having same name should have same semantics. Do you agree with that?

Let's just name it `measurement_gooid`*. I use `oo` from _OONI_ and I don't want it to be confused with 128-bit `GUID` format. OTOH, _goo-id_ is probably too self-critical. :)

What do you think?

Reply:
Actually, I was a bit wrong. We don't have thing named `measurement_id ` in th data, we have `id` and `report_id`, so we probably can take `measurement_id` name here. It may be some sort of backwards incompatibility, but I don't feel like it matters.

Reply:
TheTorProject/ooni-pipeline#83